.github/workflows/build.yml

```diff
@@ -0,0 +1,60 @@
+name: hdzero-vtx build
+on: [push, pull_request]
+jobs:
+  targets:
+    runs-on: ubuntu-latest
+    outputs:
+      targets: ${{ steps.set-targets.outputs.targets }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: set-targets
+        run: echo "::set-output name=targets::[$(grep -r "\[env:" platformio.ini | sed 's/.*:\(.*\)].*/"\1"/' | tr '\n' ',')]"
+
+  build:
+    needs: targets
+    strategy:
+      fail-fast: false
+      matrix:
+        target: ${{fromJSON(needs.targets.outputs.targets)}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v3.x
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ matrix.target }}
+
+      - name: Install PlatformIO
+        run: |
+          python -m pip install --upgrade pip
+          pip install platformio
+          pip install six
```

Comment:
Most likely this can be removed and included locally if ever needed.

Reply:
right, i got rid of it.